### ISSUE TITLE
Marriages - Notes field problems

### ISSUE THREAD

**Issue by dreams-togo** [FreeUKGen/ComETT#83]
If you try to write a longer entry then the text disappears under the next field heading box and you can&#039;t see what you are typing by any method I can find. The field should have a wrap text and expand vertically or have a much much larger box. To put the entry in I have had to type it up in word first and then copy and paste it into the box. I am reporting as an urgent bug as it makes the data field almost unusable

**Comment by dreams-togo** [comment]
LouSmi said: I too find this difficult

**Comment by DeniseColbert** [comment]
Dale will check that the testers are happy with the fix

**Comment by BevStubbs** [comment]
I have just tested the fix today. I started a new ma transcription package and entered a note. The fix worked in that I was able to  enter the infomation after I expanded the field vertically so I could see the whole entry and edit it. However a further problem arose when I clicked on submit. The notes field was blank in the verify screen even though I scrolled it's full length. Leaving it unchanged (theoretically blank) I then verifyed the record. On checking the entry on the line at the bottom of the screen the full entry was there it was just hidden in the verify record screen. A further discussion point may be whether we need the verify by line option at all.
